orchagent/bufferorch.cpp

```diff
@@ -822,11 +822,13 @@ task_process_status BufferOrch::processQueue(KeyOpFieldsValuesTuple &tuple)
                 {
                     auto flexCounterOrch = gDirectory.get<FlexCounterOrch*>();
                     auto queues = tokens[1];
-                    if (op == SET_COMMAND && flexCounterOrch->getQueueCountersState())
+                    if (op == SET_COMMAND && (flexCounterOrch->getQueueCountersState()
+                                            || flexCounterOrch->getQueueWatermarkCountersState()))
```

Comment:
```suggestion
                    if (op == SET_COMMAND &&
                         (flexCounterOrch->getQueueCountersState() || flexCounterOrch->getQueueWatermarkCountersState()))
```
Same as below.
It's better to keep the subexpressions with higher priority in one line

Reply:
fixed, I updated the PR